client/landing/gutenboarding/onboarding-block/acquire-intent/vertical-select/style.scss

```diff
@@ -1,44 +1,88 @@
 @import 'assets/stylesheets/gutenberg-base-styles';
+@import '../../../variables.scss';
 
 // Restyle `<Suggestion />` component
 .vertical-select {
+	transition: flex-grow $acquire-intent-transition-duration $acquire-intent-transition-algorithm;
+	display: flex;
+	flex-direction: column;
+	&--with-suggestions {
+		flex-grow: 1;
+	}
+
+	@include break-small {
+		display: block;
+	}
+
 	.suggestions__category-heading {
 		display: none;
 	}
+	.suggestions__title {
+		line-height: normal;
+		padding: 10px 16px;
+	}
 	.suggestions__item {
 		border: none;
 		background-color: var( --studio-white );
 
 		&.has-highlight {
-			box-shadow: inset 0 0 0 1px var( --studio-blue-30 );
+			box-shadow: none;
 			background-color: var( --studio-white );
-			border-radius: 2px;
+
+			@include break-small {
+				box-shadow: inset 0 0 0 1px var( --studio-blue-30 );
+				border-radius: 2px;
+			}
 		}
 	}
 }
 
 .vertical-select__suggestions-wrapper {
 	position: relative;
+	display: block;
+	padding-right: 20px; // prevent arrow overflow on mobile
+	flex: 1;
+
+	@include break-small {
+		display: inline;
+		padding-right: 0;
+	}
 }
 
 .vertical-select__suggestions {
 	position: absolute;
-	left: 0;
-	width: 250px;
-	max-height: 400px;
+	left: -16px;
+	margin-top: 20px;
+	min-height: 100px;
+    max-height: calc( 100% - 50px );
```

Comment:
Still whitespace issues here. Should be fixed on precommit hook, this is odd.

Reply:
Last time when I checked, prettier runs only on JS, not CSS to prevent unwanted changes. I'll look more into configuring stylelint to detect these issues 🤔